docs/notes/v6.0.0.md

```diff
@@ -0,0 +1,72 @@
+# MinIO Operator v6.0.0 release notes
+
+## About this release
+
+MinIO Operator is the production-ready easiest way to manage MinIO deployments in kubernetes. And with a new version, new features, a lot of bug fixes, new kubernetes base version requirements, deprecations and much more.
+
+## What’s new?
+
+Operator 6.0.0 is now available, this release requires Kubernetes version 1.25.0 or later, you must upgrade your Kubernetes cluster to 1.25.0 or later to use Operator v6.0.0+.
```

Comment:
@cniackz  @dvaldivia why is needed to start on Kubernetes 1.25.0 and what happens with installs on Kubernetes 1.21 or previous? Are these installs no longer supported?

Reply:
We have been encouraging users to adhere to the supported Kubernetes versions listed on the [Kubernetes releases page](https://kubernetes.io/releases/). Version 1.21 is no longer supported, so we need to upgrade to a newer, supported version based on the `End of Life` schedule.